src/systemtest/java/util/PropertiesHandler.java

```diff
@@ -0,0 +1,39 @@
+package util;
+
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.InputStream;
+import java.util.Map.Entry;
+import java.util.Properties;
+
+public class PropertiesHandler {
+
+    public static void setProperties() throws Throwable {
+        final String systemTestFilePath;
+
+        if (System.getProperty("propertiesFile") != null) {
+            systemTestFilePath = System.getProperty("propertiesFile");
+        } else {
+            systemTestFilePath = "src/test/resources/system_test.properties";
+        }
+        InputStream systemTestFileInputStream = new FileInputStream(new File(systemTestFilePath));
+        Properties systemTestProperties = new Properties();
+        systemTestProperties.load(systemTestFileInputStream);
+        for (Entry entry : systemTestProperties.entrySet()) {
+            String key = entry.getKey()
+                              .toString();
+            String valueInFile = entry.getValue()
+                                      .toString();
+            String value = getPeriorityValue(key, valueInFile);
+            System.setProperty(key, value);
+        }
+    }
+
+    public static String getPeriorityValue(String key, String value) {
+        if ((System.getProperty(key) != null) || (System.getenv(key) != null)) {
+            return (System.getenv(key) != null) ? System.getenv(key) : System.getProperty(key);
```

Comment:
You need to convert the getenv key from "_" to "." so it becomes a Java property. I guess we could just set it to System.setPortperty(),,, depending how file is handled and overided by these new environment properties values.